plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -42,26 +43,22 @@ void testShowTables()
     @Test
     void testColumnComment()
     {
-        assertUpdate("CREATE TABLE faker.default.comments (id INTEGER, name VARCHAR)");
-        assertUpdate("COMMENT ON COLUMN comments.name IS 'comment text'");
-
-        assertTableColumnNames("faker.default.comments", "id", "name");
-
-        assertUpdate("DROP TABLE faker.default.comments");
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "comment", "(id INTEGER, name VARCHAR)")) {
+            assertUpdate("COMMENT ON COLUMN %s.name IS 'comment text'".formatted(table.getName()));
+            assertQuery("SHOW COLUMNS FROM " + table.getName(), "VALUES ('id', 'integer', '', ''), ('name', 'varchar', '', 'comment text')");
+        }
     }
 
     @Test
     void testCannotCommentRowId()
     {
-        assertUpdate("CREATE TABLE faker.default.cannot_comment (id INTEGER, name VARCHAR)");
-        @Language("SQL")
-        String testQuery = "COMMENT ON COLUMN \"cannot_comment\".\"$row_id\" IS 'comment text'";
-
-        assertThatThrownBy(() -> getQueryRunner().execute(testQuery))
-                .as("Query: " + testQuery)
-                .hasMessageContaining("Cannot set comment for $row_id column");
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "cannot_comment", "(id INTEGER, name VARCHAR)")) {
+            String testQuery = "COMMENT ON COLUMN \"%s\".\"$row_id\" IS 'comment text'".formatted(table.getName());
 
-        assertUpdate("DROP TABLE faker.default.cannot_comment");
+            assertThat(query(testQuery))
+                    .nonTrinoExceptionFailure()
+                    .hasMessageContaining("Cannot set comment for $row_id column");
```

Comment:
https://github.com/trinodb/trino/blob/115b8ea7a375ba820a5ce98688594fa9f8c85489/plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java#L265-L267 should throw `TrinoException`.

Reply:
Done